nexus/db-queries/Cargo.toml

```diff
@@ -74,6 +74,7 @@ omicron-test-utils = { workspace = true, optional = true }
 [features]
 # Enable to export `TestDatabase`
 testing = ["omicron-test-utils"]
+omdb = []
```

Comment:
I'm not sure this is worth hiding behind a feature flag. If we're exposing this in the nexus we're shipping to customers, why not just "let this be part of the API" of this crate?

There are already APIs in `db-queries` which are only used by `omdb`

Reply:
Yeah, I think I was over-zealous with this. removed in 77c0cf9b6